packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1031,6 +1032,21 @@ export class ProjectService {
                       [oldDimension]: row[newDimension],
                   }))
                 : rows;
+        const exploreDimensions = getDimensions(explore);
```

Comment:
I know we also `getDimensions` when running `updateMetricQueryGranularity` in https://github.com/lightdash/lightdash/pull/8078/files - maybe we can pass the this value to `updateMetricQueryGranularity` and we run this piece of code L1035-1049 before that? what do you think @rephus ?

Reply:
yeah, feel free to change this to make it more efficient